pyvortex/src/arrays/mod.rs

```diff
@@ -220,7 +220,7 @@ impl PyArray {
         if let Some(chunked_array) = array.as_opt::<ChunkedArray>() {
             // We figure out a single Arrow Data Type to convert all chunks into, otherwise
             // the preferred type of each chunk may be different.
-            let arrow_dtype = chunked_array.dtype().to_arrow_dtype()?;
+            let arrow_dtype = chunked_array.dtype().to_arrow_field()?;
```

Comment:
Yup, this makes a huge amount of sense. Probably worth pulling this into a PR as well?